Node/Stream.fs

```diff
@@ -180,7 +180,7 @@ type [<AllowNullLiteral>] Transform<'a, 'b> =
     /// It is possible that no output is generated from any given chunk of input data.
     /// The callback function must be called only when the current chunk is completely consumed. The first argument passed to the callback must be an Error object if an error occurred while processing the input or null otherwise. If a second argument is passed to the callback, it will be forwarded on to the readable.push() method.
     /// Transform._transform() is never called in parallel; streams implement a queue mechanism, and to receive the next chunk, callback must be called, either synchronously or asynchronously.
-    abstract _transform: chunk: 'a * encoding: string * callback: (Error option * 'b option -> unit) -> unit
+    abstract _transform: chunk: 'a * encoding: string * cb: (Error option * 'b option -> unit) -> unit
```

Comment:
This should be also `cb: (Error option -> 'b option -> unit)`. See https://github.com/fable-compiler/Fable/issues/1312#issuecomment-354681614

Reply:
I'll update